CI/.kind-config.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: control-plane
+  - role: control-plane
+  - role: worker
+  - role: worker
```

Comment:
Is the kind config being used? In the action I see only one node is getting created. does this file need to be in the chaos-kubox/actions/kind? 

```
Run kubectl get nodes
  kubectl get nodes
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
NAME                 STATUS   ROLES                  AGE   VERSION
kind-control-plane   Ready    control-plane,master   79s   v1.21.1
```